src/sketch.js

```diff
@@ -230,3 +271,28 @@ function draw()
     // draw webcam input
     drawWebcamInput(3);
 }
+
+/**
+ * I change the audio pan over time.
+ *
+ * Returns:
+ *  undefined.
+ */
+function alterPan()
+{
+    // make target slightly above threshold to avoid getting stuck on extremes
+    let target = (PAN_THRESHOLD * 1.5) * pan_direction
+
+    /// interpolate previous pan value in direction of the threshold on amplitude
+    pan = lerp(pan, target, amplitude.getLevel() * PAN_STEP_FACTOR);
+
+    // when pan value is above threshold change pan direction
```

Comment:
Please use the standar format:

```suggestion
    // pan value is above threshold: change pan direction
```